modules/appeals_api/spec/workers/higher_level_review_upload_status_updater_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require AppealsApi::Engine.root.join('spec', 'support', 'shared_examples_for_monitored_worker.rb')
```

Comment:
Assuming our spec_helper file will actually load the `support` files correctly, I'd like keep our spec headers more consistent and include `spec_helper` instead of including the individual support file.